torch/csrc/jit/ir.h

```diff
@@ -163,7 +163,8 @@ struct Value {
  public:
   Value* setType(TypePtr type);
   void inferTypeFrom(const at::Tensor& output) {
-    setType(CompleteTensorType::create(output));
+    auto complete = CompleteTensorType::create(output.scalar_type(), output.options().device(), output.sizes());
```

Comment:
Why did this change? The details of how to create a CompleteTensorType from Tensor need to stay local to where it is defined, otherwise it is hard to update.

Reply:
Just because if the default create() function calls strides() on a sparse tensor it'll fail.